### ISSUE TITLE
ORM Adapter For NextAuth

### ISSUE THREAD

**Issue by Immortalin** [edgedb/edgedb#3485]
[NextAuth](https://github.com/nextauthjs/next-auth) is a popular, almost _de facto_ authentication framework for Next.js. It supports many ORM backends [via adapters](https://next-auth.js.org/adapters/overview). If EdgeDB can provide a first party backend adapter, it would help accelerate adoption of EdgeDB. The model for NextAuth is [very simple ](https://next-auth.js.org/adapters/models) and integration only takes [a few lines of code](https://github.com/nextauthjs/next-auth/blob/main/packages/adapter-prisma/src/index.ts).

**Comment by colinhacks** [comment]
Good idea! I've used next-auth several times in various side projects, should be quick to do. I'll look into it.
